Wikipedia/Localizations/qqq.lproj/Localizable.strings

```diff
@@ -294,10 +290,3 @@
 "captcha-reload" = "Title for button used to request another CAPTCHA to solve";
 "captcha-prompt" = "Text prompting user to solve a CAPTCHA";
 "mwk-empty-title-error" = "Error description for when an operation couldn't be performed since no title was specified.";
-"home-title" = "Title for home interface.";
```

Comment:
Hmm we may have a problem - I don't think the qqq entries should be being deleted...